internal/delivery/http/v1/request/matches_request.go

```diff
@@ -0,0 +1,7 @@
+package request
+
+type CreateMatch struct {
+	MatchCatID string `json:"matchCatId" validate:"required"`
+	UserCatID  string `json:"userCatId" validate:"required"`
+	Message    string `json:"message" validate:"required"`
```

Comment:
Ini udah pake `validate` cuman kayaknya belum pake `validator` di `create_match_handler`-nya ya? sama kayaknya bisa tambahin validasi `min` sama `max` char:

```suggestion
	Message    string `json:"message" validate:"required,min=5,max=120"`
```

Reply:
adjusted mas thanks